content/css/sel4.css

```diff
@@ -286,3 +286,61 @@ a.divLink {
   font-size:20px;
   font-weight: bold;
 }
+
+.news {
+  width: auto;
+  height:auto;
+  padding-top:5px;
+  padding-left:10px;
+  padding-right:5px;
+  margin-top:20px;
+  margin-left:60px;
+  margin-right:60px;
+  border-radius: 10px;
+  background-color: #e5e9ee; /* ligher than hover D6DBE4 */
+}
+
+.news:hover {
+  background-color: #D6DBE4; /* neutralLighter */
+}
+
+.news-column {
+  float: left;
+  padding: 5px;
+}
+
+.news-clearfix::after {
+  content: "";
+  clear: both;
+  display: table;
+}
+
+.news-title {
+  width: 75px;
+  font-size: 200%;
+  font-family: courier;
+  line-height: 80%;
+}
+
+.news-content {
+  width: auto;
+  font-family: courier;
+}
+
+
+.news-item {
+  width: auto;
+  height:auto;
+  padding:5px;
+  margin-top:20px;
+  margin-bottom:5px;
```

Comment:
Also possibly better with `em`/`ex` instead of pixels. But I'm not sure, best way is to try it out for different sizes (font and window sizes)

Reply:
General observation (not tied to this change): The CSS file takes particularly long to load ATM, instead of a few seconds it takes minutes until the page formats correctly

Reply:
OK, this comment was misplaced, sorry

Reply:
done.